test/core/transport/binder/end2end/fuzzers/BUILD

```diff
@@ -0,0 +1,100 @@
+# Copyright 2021 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//bazel:grpc_build_system.bzl", "grpc_cc_library", "grpc_package")
+load("//test/core/util:grpc_fuzzer.bzl", "grpc_proto_fuzzer")
+
+grpc_package(
+    name = "test/core/transport/binder/end2end/fuzzers",
+    features = [
+        "layering_check",
+    ],
+)
+
+licenses(["notice"])
+
+# Defines protobuf messages for generating inputs. We manually define proto
+# library rules here so the same proto file can be shared between several
+# grpc_proto_fuzzer targets
+proto_library(
+    name = "binder_transport_fuzzer_proto",
+    srcs = ["proto/binder_transport_fuzzer.proto"],
+)
+
+cc_proto_library(
+    name = "binder_transport_fuzzer_cc_proto",
+    deps = ["binder_transport_fuzzer_proto"],
+)
```

Comment:
Self note: This fails copybara import "name 'cc_proto_library' is not defined"

Probably need to use `grpc_proto_library` here

Reply:
Done